### PR TITLE
Update Swivel sample and Matching Engine sample

### PR DESCRIPTION
**Pull request by yinghsienwu**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

### CODE REVIEW

**Comment by yinghsienwu** [comment]
Logs: https://pantheon.corp.google.com/cloud-build/builds/030841be-adef-4e87-b903-bf45c38bb70a?project=python-docs-samples-tests

Detected the following error in intro-swivel.ipynb:
Step #3: ContextualVersionConflict: (google-api-core 2.3.2 (/builder/home/.local/lib/python3.9/site-packages), Requirement.parse('google-api-core<2dev,>=1.21.0'), {'google-api-python-client'})

**Comment by yinghsienwu** [comment]
In summary, pinned KFP version caused the version conflict.  The execution test passed when I used the Vertex SDK to submit a Vertex Pipeline job and did not specify the service account.

File: notebooks/official/matching_engine/intro-swivel.ipynb [hunk 1]
```diff
@@ -153,7 +153,7 @@
       "source": [
```
Comment (andrewferlitsch) [inline notebooks/official/matching_engine/intro-swivel.ipynb:153]: <p>network_name: my_network_name -&gt; network_name: e.g., my_network_name </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (andrewferlitsch) [inline notebooks/official/matching_engine/intro-swivel.ipynb:153]: <p>I am confused. Subtitle is about Tensorboard, but text refers to Vertex ML Metadata browser.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (yinghsienwu) [reply notebooks/official/matching_engine/intro-swivel.ipynb:153]: done

Comment (yinghsienwu) [reply notebooks/official/matching_engine/intro-swivel.ipynb:153]: Good call. I updated the description.  Thanks!

Comment (andrewferlitsch) [reply notebooks/official/matching_engine/intro-swivel.ipynb:153]: <p>close.</p><p>network_name: network_name: e.g -&gt; network_name: e.g.,</p><p> </p><p>Note you repeated network_name twice.</p>

Comment (yinghsienwu) [reply notebooks/official/matching_engine/intro-swivel.ipynb:153]: done

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**